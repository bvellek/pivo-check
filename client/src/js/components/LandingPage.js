import React from 'react';
import Footer from './Footer';

const LandingPage = () => (
  <div className="landing-page">
    <section className="landing-callout">
      <div className="callout-text">
        <p>Drink<span className="teal-text">*</span> your way through town. Check off the breweries as you&nbsp;go!</p>
        <p><span className="teal-text">*</span>Please do it responsibly… and use <nobr>PIVO-CHECK</nobr> to keep track of all the breweries you&nbsp;visit.</p>
      </div>
      <img className="callout-img" src="./img/map.svg" alt="Cartoon map of breweries" />
    </section>

    <main role="main">
      <section className="landing-description">
        <h2>How to get started...</h2>
        <p>To start your journey register and simply add a city to your list. You will get a list of all the breweries ‘Open to the Public’ in the city. Visit a brewery and check it off your&nbsp;list.</p>
      </section>

      <section className="landing-cta">
        <h2>
          <a href="#">Login/Register</a>
        </h2>
      </section>
    </main>

    <Footer />
  </div>
);

export default LandingPage;
