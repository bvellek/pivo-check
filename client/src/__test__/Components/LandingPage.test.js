import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LandingPage from '../../js/components/LandingPage';

describe('<LandingPage />', () => {
  it('renders without crashing', () => {
    shallow(<LandingPage />);
  });
  it('it renders correctly', () => {
    const component = shallow(
      <LandingPage />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
