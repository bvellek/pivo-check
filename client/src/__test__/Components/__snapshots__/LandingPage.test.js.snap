// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> it renders correctly 1`] = `
<div>
  <LandingHeader />
  <LandingContent />
</div>
`;
